en/flight_controller/raspberry_pi_navio2.md

```diff
@@ -26,22 +26,29 @@ By upgrading, a new kernel can get installed which lacks the necessary HW suppor
 
 ## Setting up Access
 
-The Raspbian image has SSH setup already.
+The Raspberry Pi OS image has SSH setup already.
 Username is "pi" and password is "raspberry".
-You can connect to your RPi2/3 over a network (Ethernet is set to come up with DHCP by default) and then proceed to configure WiFi access.
 We assume that the username and password remain at their defaults for the purpose of this guide.
 
-To setup the RPi2/3 to join your local wifi, follow [this guide](https://www.raspberrypi.org/documentation/configuration/wireless/wireless-cli.md).
+To setup the Pi to join your local wifi, follow [this guide](https://www.raspberrypi.org/documentation/configuration/wireless/wireless-cli.md), or connect it via an ethernet cable.
 
-Find the IP address of your Pi from your network, and then you can proceed to connect to it using SSH.
+To connect to your pi via SSH, use the default username(pi) and hostname(navio). Alternatively if this doesn't work, you can find the IP address of your Pi and specify it.
```

Comment:
```suggestion
To connect to your RPi via SSH, use the default username (`pi`) and hostname (`navio`).
Alternatively (if this doesn't work), you can find the IP address of your RPi and specify it.
```

Reply:
I agree with one minor change, for most of the guide the Raspberry Pi is just referred to as a Pi, RPi is fine too just want to make sure it's consistent.

Reply:
@johnBrereton Sorry, missed this comment when I merged this. But I had already checked and seen you made the changes consistently, so this remains Pi.